### PR TITLE
QUEST_LV_0300 625 ~ 1000

### PR DESCRIPTION
**Pull request by Requinn**
More cleaning.

### CODE REVIEW

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 1]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
-QUEST_LV_0300_20151022_000654	Who are you? Is this your first time?{nl}Have you gotten the permission from the village headman?    
-QUEST_LV_0300_20151022_000655	The Revelator from the folklore...{nl}I've heard it from the priest of the village. You saved him using some kind of a ray, right?{nl}That ray... I want to see it one day too!
-QUEST_LV_0300_20151022_000656	Ah.. After setting up the sculpture, those Ferrets became more violent?{nl}That's probably because the sculpture was not completed yet...{nl}I feel sorry for Widas...
-QUEST_LV_0300_20151022_000657	Oh... Hello...{nl}I am sorry, but if you don't have any business here, please leave now...{nl}Right now, the atmosphere in this town is bad...
-QUEST_LV_0300_20151022_000658	Revelator! Are you the Revelator from the folklore who will save the village with the ray?{nl}I saw it. Your splendid ray...{nl}If it wasn't for that ray, I would have already been attacked by those Ferrets.{nl}Thank you so much.
+QUEST_LV_0300_20151022_000653	Let's go see if those Ferrets ate the beef jerky...{nl}Would it be dangerous?{nl}I am curious...
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Ferrets -> ferrets

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 2]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
-QUEST_LV_0300_20151022_000662	...No... This is not it...
+QUEST_LV_0300_20151022_000661	Revelator... Revelator... No... our village is already done...{nl}You came too late... it won't make any difference now...{nl}Our village was already done for when Goddess Lada was captured...
+QUEST_LV_0300_20151022_000662	...No... This isn't it...
 QUEST_LV_0300_20151022_000663	It's so good to hear that you've come here to save the village.{nl}Welcome. I am Dievdirbys Widus.{nl}I am creating sculptures to turn those Ferrets back to what they used to be.
-QUEST_LV_0300_20151022_000664	The Ferrets have become more violent... I feel like I've made demonic totems..{nl}I guess I better create from scratch...
-QUEST_LV_0300_20151022_000665	Uh? Revelator... Why have you come so early?{nl}I will take a look at this place, so please help the villagers first.
+QUEST_LV_0300_20151022_000664	The Ferrets have become more violent... I feel like I've made demonic totems...{nl}I guess I better create new ones from scratch...
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Ferrets -> ferrets

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 3]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
+QUEST_LV_0300_20151022_000665	Uh? Revelator... Why did you arrive so early...{nl}I will take a look at this place, so please help the villagers first.
 QUEST_LV_0300_20151022_000666	Please follow the girl and save the villagers and Goddess Lada...
-QUEST_LV_0300_20151022_000667	I knew that you were going to do it!{nl}Thank you. Thank you so much.{nl}I want to appreciate Goddess Laima who guided you to this place.
-QUEST_LV_0300_20151022_000668	Thank you for saving this village.{nl}However, those Ferrets are still going crazy, so I better make the sculpture quickly.
+QUEST_LV_0300_20151022_000667	I knew that you could do it!{nl}Thank you. Thank you so much.{nl}I want to thank Goddess Laima who guided you to this place.
+QUEST_LV_0300_20151022_000668	Thank you for saving this village.{nl}However, since those Ferrets are still going crazy, I better make the sculpture quickly.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Ferrets -> ferrets

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 4]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
-QUEST_LV_0300_20151022_000670	That is so amazing that you managed to defeat the Demon Lord!{nl}Because of you, the Ferrets don't steal our food anymore.{nl}Thank you.
-QUEST_LV_0300_20151022_000671	Thanks. I am grateful of Goddess Laima who has brought you here.{nl}You are our blessing to our village.
-QUEST_LV_0300_20151022_000672	I hope those Ferrets gain consciousness soon...
-QUEST_LV_0300_20151022_000673	What if those Ferrets have that mind and attack us...{nl}We would not be able to avoid the fight... Right?
-QUEST_LV_0300_20151022_000674	Thank you for saving this village.{nl}You have defeated the Demon Lord and rescued the goddess.{nl}Now, we can concentrate on the Ferrets.{nl}Thank you from the bottom of my heart for reducing the worries of this village.
+QUEST_LV_0300_20151022_000670	That is so amazing that you managed to defeat the Demon Lord!{nl}Because of you, the Ferrets won't steal our food anymore.{nl}Thank you.
+QUEST_LV_0300_20151022_000671	Thanks. I am grateful of Goddess Laima who has brought you here.{nl}You are a blessing to our village.
+QUEST_LV_0300_20151022_000672	I hope those Ferrets get a hold of themselves soon...
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Ferrets -> ferrets

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 5]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
+QUEST_LV_0300_20151022_000673	If those Ferrets are attacking us with that in mind...{nl}We wouldn't need to fight... Right?
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Ferrets -> ferrets

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 6]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
+QUEST_LV_0300_20151022_000668	Thank you for saving this village.{nl}However, since those Ferrets are still going crazy, I better make the sculpture quickly.
 QUEST_LV_0300_20151022_000669	Thank you for saving the village and Goddess Lada.
-QUEST_LV_0300_20151022_000670	That is so amazing that you managed to defeat the Demon Lord!{nl}Because of you, the Ferrets don't steal our food anymore.{nl}Thank you.
-QUEST_LV_0300_20151022_000671	Thanks. I am grateful of Goddess Laima who has brought you here.{nl}You are our blessing to our village.
-QUEST_LV_0300_20151022_000672	I hope those Ferrets gain consciousness soon...
-QUEST_LV_0300_20151022_000673	What if those Ferrets have that mind and attack us...{nl}We would not be able to avoid the fight... Right?
-QUEST_LV_0300_20151022_000674	Thank you for saving this village.{nl}You have defeated the Demon Lord and rescued the goddess.{nl}Now, we can concentrate on the Ferrets.{nl}Thank you from the bottom of my heart for reducing the worries of this village.
+QUEST_LV_0300_20151022_000670	That is so amazing that you managed to defeat the Demon Lord!{nl}Because of you, the Ferrets won't steal our food anymore.{nl}Thank you.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Ferrets -> ferrets

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 7]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
-QUEST_LV_0300_20151022_000644	The act of awakening the dead and disturbing their eternal sleep is what they call the truth?
-QUEST_LV_0300_20151022_000645	I was managing the lands of the kingdom when I was alive and they call me a sinner?
-QUEST_LV_0300_20151022_000646	Ayaya... The toe nails of those Ferrets are so sharp...
-QUEST_LV_0300_20151022_000647	The villagers here are not fond of outsiders.{nl}So, you better not expect any kindness from them.
+QUEST_LV_0300_20151022_000642	Those who believe in the goddesses are fools. {nl}The Divine Tree, which still stands in the kingdom's capital, will save you.
+QUEST_LV_0300_20151022_000643	You should't want salvation that easily.{nl}Think about what you've done for the Divine Tree before you request something from the Divine Tree.
+QUEST_LV_0300_20151022_000644	The act of awakening the dead and disturbing their eternal rest is what they call the truth?
+QUEST_LV_0300_20151022_000645	While I was alive, I managed the lands of the kingdom, yet they called me someone guilty of high treason.
+QUEST_LV_0300_20151022_000646	Ayaya... The claws of those Ferrets are so sharp...
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Ferrets -> ferrets

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 8]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
-QUEST_LV_0300_20151022_000657	Oh... Hello...{nl}I am sorry, but if you don't have any business here, please leave now...{nl}Right now, the atmosphere in this town is bad...
-QUEST_LV_0300_20151022_000658	Revelator! Are you the Revelator from the folklore who will save the village with the ray?{nl}I saw it. Your splendid ray...{nl}If it wasn't for that ray, I would have already been attacked by those Ferrets.{nl}Thank you so much.
+QUEST_LV_0300_20151022_000653	Let's go see if those Ferrets ate the beef jerky...{nl}Would it be dangerous?{nl}I am curious...
+QUEST_LV_0300_20151022_000654	Who are you? Is this your first time here?{nl}Have you gotten the permission from the village headman?    
+QUEST_LV_0300_20151022_000655	The Revelator from folklore...{nl}I've heard it from the village priest. You saved him using some kind of light, right?{nl}That light... I want to see it one day too!
+QUEST_LV_0300_20151022_000656	Ah... After setting up the sculpture, those Ferrets became more violent?{nl}That's probably because the sculpture wasn't completed yet...{nl}My apologies Widas...
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Ferrets -> ferrets

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 9]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
+QUEST_LV_0300_20151022_000671	Thanks. I am grateful of Goddess Laima who has brought you here.{nl}You are a blessing to our village.
+QUEST_LV_0300_20151022_000672	I hope those Ferrets get a hold of themselves soon...
+QUEST_LV_0300_20151022_000673	If those Ferrets are attacking us with that in mind...{nl}We wouldn't need to fight... Right?
+QUEST_LV_0300_20151022_000674	Thank you for saving this village.{nl}You have defeated the Demon Lord and rescued the goddess.{nl}Now, we can concentrate on the Ferrets.{nl}I thank you from the bottom of my heart for reducing the worries of this village.
 QUEST_LV_0300_20151022_000675	Thank you for defeating the Demon Lord and saving the goddess.
-QUEST_LV_0300_20151022_000676	Since the goddesses have come back now, we will rebuild the shrine with our villagers.
-QUEST_LV_0300_20151022_000677	Thank you for resolving the disaster of the village.{nl}I underestimated you... And for that, I apologize...
+QUEST_LV_0300_20151022_000676	Since the goddesses has come back now, we will rebuild the shrine with our villagers.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Goddesses has?

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 10]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
-QUEST_LV_0300_20151224_000689	I've heard that there were soldiers who lost their contact {nl}at the training field of conflict on Medzio Diena.{nl}We didn't expect them to be alive though...{nl}
-QUEST_LV_0300_20151224_000690	At least we will be able to remember the loyalty the army had dedicated to the kingdom.
+QUEST_LV_0300_20151102_000682	Hurk... Don't talk to me.{nl}I feel like throwing up...
+QUEST_LV_0300_20151102_000683	I shouldn't lie around in a time like this...
+QUEST_LV_0300_20151224_000684	Of course I won!{nl}However, I said I dropped the necklace I recieved as evidence from the guy into the sea.{nl}
+QUEST_LV_0300_20151224_000685	Back then, I gave up because the sea was too deep...{nl}Even now I'm not sure if I can still find it.
+QUEST_LV_0300_20151224_000686	If I can find it, I think I will be able to deal a blow to those who says I am a bluff.{nl}It's near Kranto Coast. I recognized you as a Corsair, so you should be able to find it quickly.
+QUEST_LV_0300_20151224_000687	The Centurion knows how to dedicate himself for the kingdom...{nl}He should know its honor.{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: A Centurion

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 11]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
+QUEST_LV_0300_20151224_000688	I am looking for the flags of a few soldiers who were annihilated on Medzio Diena.{nl}I want you to help me with this, for the honor of these soldiers.
+QUEST_LV_0300_20151224_000689	I've heard that there were soldiers who lost contact at the combat training field on Medzio Diena.{nl}We didn't expect them to be alive though...{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Training field of conflict = map. Check ETC for the correct name.

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 12]
```diff
@@ -879,46 +879,46 @@ QUEST_LV_0300_20151224_000878	Timing Attacks [Fencer Advancement]
...
-QUEST_LV_0300_20151224_000885	Defeat the monster to obtain the relic that possesses the regret
-QUEST_LV_0300_20151224_000886	The Necromancer Master wants to obtain the relic that possesses the strong grudge. Defeat the monsters at Videntis Shrine to obtain the relic that possesses the strong regret.
+QUEST_LV_0300_20151224_000882	Defeat monsters and obtain a piece of Legwyn's seal
+QUEST_LV_0300_20151224_000883	The Corsair Master wants to obtain the Great Pirate Vladislovas's necklace, which he had won before and used as a proof of victory. Look for the necklace at Krantos Coast, which used to be a sea.
+QUEST_LV_0300_20151224_000884	The Doppelsoeldner Master told you that he lost contact with the envoy from Pasaka Plateau that he was to meet. Look for the whereabouts of the envoy to find out what happened.
+QUEST_LV_0300_20151224_000885	Defeat the monster to obtain the relic that possesses a strong grudge
+QUEST_LV_0300_20151224_000886	The Necromancer Master wants to obtain the relic that possesses the strong grudge. Defeat the monsters at Videntis Shrine to obtain the relic that possesses a strong grudge.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: a strong grudge

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 13]
```diff
@@ -946,7 +946,7 @@ QUEST_LV_0300_20151224_000945	You've gained enough experience on Poison and Slee
 QUEST_LV_0300_20151224_000946	Talk with the Kabbalist Master
 QUEST_LV_0300_20151224_000947	Talk with the Kabbalist Master at Istora Ruins.
 QUEST_LV_0300_20151224_000948	Defeat the monster indicated by the number displayed on the crystal sphere
-QUEST_LV_0300_20151224_000949	The Kabbalist Master told you that in order to become a Kabbalist, you should be able to undertand all numbers that are hidden in all properties in the world. Use the crystal sphere that has appeared at Cranto Coast and defeat the monster which the number on the sphere indicates.
+QUEST_LV_0300_20151224_000949	The Kabbalist Master told you that in order to become a Kabbalist, you should be able to undertand all the numbers hidden in all properties of the world. Use the crystal sphere that has appeared at Cranto Coast and defeat the monster which the number on the sphere indicates.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: undertand -> understand

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 14]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
-QUEST_LV_0300_20151022_000662	...No... This is not it...
-QUEST_LV_0300_20151022_000663	It's so good to hear that you've come here to save the village.{nl}Welcome. I am Dievdirbys Widus.{nl}I am creating sculptures to turn those Ferrets back to what they used to be.
-QUEST_LV_0300_20151022_000664	The Ferrets have become more violent... I feel like I've made demonic totems..{nl}I guess I better create from scratch...
-QUEST_LV_0300_20151022_000665	Uh? Revelator... Why have you come so early?{nl}I will take a look at this place, so please help the villagers first.
+QUEST_LV_0300_20151022_000661	Revelator... Revelator... No... our village is already done...{nl}You came too late... it won't make any difference now...{nl}Our village was already done for when Goddess Lada was captured...
+QUEST_LV_0300_20151022_000662	...No... This isn't it...
+QUEST_LV_0300_20151022_000663	It's so good to hear that you've come here to save the village.{nl}Welcome. I am Dievdirbys Widus.{nl}I am creating sculptures to turn those ferrets back to what they used to be.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Widus -> Widas

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 15]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
+QUEST_LV_0300_20151102_000682	Hurk... Don't talk to me.{nl}I feel like throwing up...
+QUEST_LV_0300_20151102_000683	I shouldn't lie around in a time like this...
+QUEST_LV_0300_20151224_000684	Of course I won!{nl}However, I said I dropped the necklace I recieved as evidence from the guy into the sea.{nl}
+QUEST_LV_0300_20151224_000685	Back then, I gave up because the sea was too deep...{nl}Even now I'm not sure if I can still find it.
+QUEST_LV_0300_20151224_000686	If I can find it, I think I will be able to deal a blow to those who says I am a bluff.{nl}It's near Kranto Coast. I recognized you as a Corsair, so you should be able to find it quickly.
+QUEST_LV_0300_20151224_000687	A Centurion knows how to dedicate himself for the kingdom...{nl}He should know its honor.{nl}
+QUEST_LV_0300_20151224_000688	I am looking for the flags of a few soldiers who were annihilated on Medzio Diena.{nl}I want you to help me with this, for the honor of these soldiers.
+QUEST_LV_0300_20151224_000689	I've heard that there were soldiers who lost contact at the Training Field of Conflict on Medzio Diena.{nl}We didn't expect them to be alive though...{nl}
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Training Field of Conflict -> Drill Ground of Confliction

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 16]
```diff
@@ -623,100 +623,100 @@ QUEST_LV_0300_20151022_000622	Although the kingdom has become ineffective, we ar
...
+QUEST_LV_0300_20151022_000645	While I was alive, I managed the lands of the kingdom, yet they called me someone guilty of high treason.
+QUEST_LV_0300_20151022_000646	Ayaya... The claws of those ferrets are so sharp...
+QUEST_LV_0300_20151022_000647	The villagers here are not fond of outsiders.{nl}Would it be better to not ask for their favor?
 QUEST_LV_0300_20151022_000648	I've never seen you before... Who are you?
 QUEST_LV_0300_20151022_000649	The Revelator from the folklore has appeared to save us...{nl}Thank you... Everyone was waiting for you...
-QUEST_LV_0300_20151022_000650	!... Why are you still here...?{nl}You better not miss the girl.{nl}Go after her.
-QUEST_LV_0300_20151022_000651	Who are you? I have never seen you around here.{nl}I am tired of those villagers, so if you don't want anything from me,{nl}please don't talk to me...
+QUEST_LV_0300_20151022_000650	Uh-huh... Why are you still here...?{nl}You better not miss the girl.{nl}Go after her.
+QUEST_LV_0300_20151022_000651	Who are you? I've never seen you around here.{nl}Even with just the village men its tiring, if you don't have anything to do can you tell them to stop talking...?
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: its -> it's

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**